common/src/main/java/com/microsoft/identity/common/internal/cache/MicrosoftStsAccountCredentialAdapter.java

```diff
@@ -208,19 +206,8 @@ public RefreshToken asRefreshToken(com.microsoft.identity.common.internal.provid
         refreshTokenOut.setTarget(refreshTokenIn.getTarget());
         refreshTokenOut.setCachedAt(String.valueOf(TimeUnit.MILLISECONDS.toSeconds(System.currentTimeMillis())));
         refreshTokenOut.setExpiresOn(refreshTokenIn.getExpiresOn());
-        //refreshTokenOut.setClientInfo(""); TODO OK to drop?
+        refreshTokenOut.setClientInfo(refreshTokenIn.getClientInfo().getRawClientInfo());
```

Comment:
Will refreshTokenIn.getClientInfo() always not null?

Reply:
It will be null only if ADFS v2 is used - @shoatman [has made changes inside of ADAL to block common cache usage if ADFS is used](https://github.com/AzureAD/azure-activedirectory-library-for-android/blob/dev/adal/src/main/java/com/microsoft/aad/adal/TokenCacheAccessor.java#L280)